src/liftwing_api/models/revertrisk.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+from liftwing_api.models.liftwing_model import LiftwingModel
+from typing import Dict
+
+class RevertRiskAPIModel(LiftwingModel):
+    def __init__(self, base_url="https://api.wikimedia.org/service/lw/inference/v1/models/revertrisk-language-agnostic:predict"):
+        super().__init__(base_url)
+
+    def request(self, payload: Dict[str, int], method: str = "POST", headers: Dict[str, str] = None) -> Dict[str, int]:
```

Comment:
nit:
The `Dict[str, int]` part is not correct as the key will indeed be a string but the values can be of any type (str or int at the moment. The same stands for the return type( the part after the arrow `->`)
```
    def request(self, payload: Dict[str, int], method: str = "POST", headers: Dict[str, str] = None) -> Dict[str, int]:
```

I would recommend that we specify `Any` for now and we look into it in the future by specifying specific schemas.
Especially the type of the return is really complex as it is custom to the model server.
import `Any` from typing:

```
from typing import Any, Dict
```
and then we would have:
```suggestion
    def request(self, payload: Dict[str, Any], method: str = "POST", headers: Dict[str, str] = None) -> Dict[str, Any]:
```

Reply:
updated